demo-kotlin/app/src/main/res/layout/activity_main.xml

```diff
@@ -25,39 +25,46 @@
     tools:context=".MainActivity">
 
   <Button
-    android:id="@+id/programmatic_autocomplete_button"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:layout_margin="@dimen/spacing_small"
-    android:text="@string/programmatic_autocomplete_geocoding"/>
+      android:id="@+id/autocomplete_button"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_margin="@dimen/spacing_small"
+      android:text="@string/autocomplete_button"/>
 
   <Button
       android:id="@+id/autocomplete_address_button"
-      android:layout_width="wrap_content"
+      android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:layout_margin="@dimen/spacing_small"
       android:text="@string/autocomplete_address_button"/>
 
   <Button
-      android:id="@+id/autocomplete_button"
-      android:layout_width="wrap_content"
+    android:id="@+id/programmatic_autocomplete_button"
```

Comment:
The indentation here seems to be a bit off. I generally just use the auto-formatter in Android Studio when working on layouts, which should format things appropriately.